src/models/user.js

```diff
@@ -4,6 +4,8 @@ const bcrypt = require('bcryptjs');
 const userSchema = mongoose.Schema({
   username: { type: String, required: true, index: { unique: true, dropDups: true } },
   password: { type: String, required: true },
+  resetPasswordToken: { type: String },
+  resetPasswordExpires: { type: Number },
```

Comment:
As this is an expiration, a date would be more appropriate for this property's schema type. This type of better optimized for indexing and will enable you to enter a human readable ISO date when setting the expiration of a field.